lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -221,6 +222,11 @@ IncrementalJIT::getSymbolAddressWithoutMangling(llvm::StringRef Name,
     if (RuntimeDyld::SymbolInfo SymInfo = m_ExeMM->findSymbol(Name))
       return llvm::orc::JITSymbol(SymInfo.getAddress(),
                                   llvm::JITSymbolFlags::Exported);
+#ifdef LLVM_ON_WIN32
+    if (const void* Sym = platform::DLSym(Name))
```

Comment:
Sorry no info on timing.  I'm not sure llvm::DynLibraries are being used anymore, and would like to keep it that way.  Both because it can't support the 'permanent' flag or unloading.  I have a subsequent commit that implements the 'permanent' flag as well as the following behavior:

```
Interpreter parent;
{
 Interpreter child(parent);
 // Load dynamic libraries
}
// ~child will unload all libraries it loaded
```
This is important to me to reclaim memory that won't be accessed anymore.